tensorflow/compiler/xla/stream_executor/cuda/BUILD

```diff
@@ -152,6 +152,7 @@ tf_cuda_cc_test(
     srcs = ["cuda_driver_test.cc"],
     tags = tf_cuda_tests_tags() + [
         "no_cuda_asan",  # TODO(b/171512140): re-enable.
+        "no_rocm",
```

Comment:
@i-chaochen saw this one in the upstream CI the other day. I'm still curious why all of sudden this test is being executed. Fine for now though.

Reply:
Actually, it has been executed all along and silently passing.

But they added a dependency which finally broke it.

I did not have time to figure out a better way to address this but we obviously should not be executing any unit tests here which implies that there are other tests which are still being run on ROCm.